src/Plugin/resource/Field/ResourceFieldEntity.php

```diff
@@ -140,6 +145,183 @@ class_implements($class_name)
   /**
    * {@inheritdoc}
    */
+  public function value(DataSourceInterface $source) {
+    // Return if this field is a callback.
```

Comment:
not clear - if this is a callback then why is it getting into ``ResourceFieldEntity``?